daemons/gptp/common/ptp_message.cpp

```diff
@@ -1502,12 +1510,23 @@ void PTPMessagePathDelayRespFollowUp::processMessage(IEEE1588Port * port)
 	    Timestamp prev_peer_ts_theirs;
 	    FrequencyRatio rate_offset;
 	    if( port->getPeerOffset( prev_peer_ts_mine, prev_peer_ts_theirs )) {
+		    FrequencyRatio upper_limit, lower_limit;
+		    upper_limit = UPPER_FREQ_LIMIT*2;
+		    upper_limit /= US_PER_SEC;
+		    upper_limit += 1;
+		    lower_limit = UPPER_FREQ_LIMIT*2;
+		    lower_limit /= US_PER_SEC;
+		    lower_limit += 1;
+
 			mine_elapsed =  TIMESTAMP_TO_NS(request_tx_timestamp)-TIMESTAMP_TO_NS(prev_peer_ts_mine);
 			theirs_elapsed = TIMESTAMP_TO_NS(remote_req_rx_timestamp)-TIMESTAMP_TO_NS(prev_peer_ts_theirs);
 			theirs_elapsed -= port->getLinkDelay();
 			theirs_elapsed += link_delay < 0 ? 0 : link_delay;
 			rate_offset =  ((FrequencyRatio) mine_elapsed)/theirs_elapsed;
-			port->setPeerRateOffset(rate_offset);
+
```

Comment:
Given the above calculation of upper_limit and lower_limit, it seems we're never going to set the rate offset, since upper_limit and lower_limit are always the same value.